test/functionalTest/cases/3727_avoid_triggering_notification_if_metadata_changes_but_not_the_attribute_value_itself/avoid_triggering_notification_if_metadata_changes_but_not_the_attribute_value_itself.test

```diff
@@ -0,0 +1,506 @@
+# Copyright 2022 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+avoid_triggering_notification_if_metadata_changes_but_not_the_attribute_value_itself
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. POST /v2/entities, to create Room1 with attribute and metadata.
+# 02. POST /v2/subscriptions, to create subscription with notifyOnMetadataChange=false.
+# 03. PATCH /v2/entities/Room1/attrs, to update the value of metadata.
+# 04. Dump the accumulator, see no notification when metadata changes.
+# 05. PATCH /v2/entities/Room1/attrs, to update the value of attribute.
+# 06. Dump the accumulator, see the notification when value of attribute changes.
+# 07. POST /v2/entities, to create Room2 with attribute and metadata.
+# 08. POST /v2/subscriptions, to create subscription with notifyOnMetadataChange=true.
```

Comment:
```suggestion
# 08. POST /v2/subscriptions, to create subscription for Room2 with notifyOnMetadataChange=true.
```

Also in echo message and expectations

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4090/commits/9fbcd9cec499f0dbadee3a2a7a8985f2fd3e9c2b